### PR TITLE
Revisions to address reviewer comments

### PR DESCRIPTION
**Pull request by cgreene**
I started revisions to address reviewer comments here. I have a few more points I'd like to make in that new section. I will attempt to wrap these up tomorrow, but I'm filing this PR in case you want to take a look through in advance 😁

### CODE REVIEW

File: content/25.discussion.md [hunk 1]
```diff
@@ -1,37 +1,45 @@
 ## Discussion
 
-A major challenge that we faced in carrying out this work was to narrow down geographic origins for some groups of names.
+There are significant technical and ethical challenges that one faces in carrying out retrospective work to examine the fairness of scientific practices.
+A major technical challenge was to narrowing down geographic origins for some groups of names.
```
Comment (trangdata) [inline content/25.discussion.md]:
```suggestion
A major technical challenge was to narrow down geographic origins for some groups of names.
```

File: content/25.discussion.md [hunk 2]
```diff
@@ -1,37 +1,45 @@
...
 Some name origin groups, such as Hispanic names, are geographically disparate.
```
Comment (trangdata) [inline content/25.discussion.md]:
```suggestion
Specific name origin groups, such as Hispanic names, are geographically disparate.
```

File: content/25.discussion.md [hunk 3]
```diff
@@ -1,37 +1,45 @@
...
 Discrepancies in representation between these groups are thus undetectable by our classifier.
 Honoree counts of those with Hispanic names are influenced from Spain as well as Latin America.
 In such cases, our analyses may substantially understate the extent to which minoritized scientists are underrepresented among honorees and authors.
-
-Biases in authorship practices may also result in our underestimation of the composition of minoritized scientists within the field.
-We estimated the composition of the field using last author status, but in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956] women are underrepresented among such authors.
+Another technical challenge is that supervised machine learning approaches are neither error free nor bias free.
+By integrating different lines of evidence and preserving uncertainty by analyzing prediction probabilities rather than applying a hard assignment for each prediction, we aimed to alleviate method-specific biases and discover insightful findings that correctly reflect the current representation diversity at conferences.
```
Comment (trangdata) [inline content/25.discussion.md:11]: 💯

File: content/25.discussion.md [hunk 4]
```diff
@@ -1,37 +1,45 @@
...
+A key element is then implementing processes to use these plans to continuously evaluate practices.
+
+A challenge that straddles the ethical and technical divide for studies examining the representation of honorees is that the background that can be best assessed, which we assess here, is the current field composition.
+Scientific societies exist to promote the discipline, and many, including ISCB, include diversity as a value.
+In these cases the ideal background distribution would be the portion of the population that would be senior computational biologists in the absence of systemic barriers to participation.
+The implication would be that scientific societies exist to reflect what the field could be, not just what it is.
+However, we are limited to measuring the field as it is.
+Furthermore, authorships, which we use to assess the field's composition, are also affected by systemic barriers to participation.
+We estimated the composition of the field using last author status, but in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956] women are underrepresented in this position.
```
Comment (trangdata) [inline content/25.discussion.md]:
```suggestion
We estimated the composition of the field using last author status, but in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956], women are underrepresented in this position.
```

File: content/25.discussion.md [hunk 5]
```diff
@@ -1,37 +1,45 @@
...
+
+A key ethical challenge with retrospective work to examine disparities is that algorithmic approaches to infer characteristics are often required.
+This leads to significant limitations such as considering gender as a binary variable.
+While this situation limits retroactive examination, there is a substantial opportunity for scientific societies, grant-making organizations, publishers, and others to proactively collect self-identified demographic information.
+A key element is then implementing processes to use these plans to continuously evaluate practices.
+
+A challenge that straddles the ethical and technical divide for studies examining the representation of honorees is that the background that can be best assessed, which we assess here, is the current field composition.
+Scientific societies exist to promote the discipline, and many, including ISCB, include diversity as a value.
+In these cases the ideal background distribution would be the portion of the population that would be senior computational biologists in the absence of systemic barriers to participation.
```
Comment (trangdata) [inline content/25.discussion.md]:
```suggestion
In these cases, the ideal background distribution would be the population of senior computational biologists in the absence of systemic barriers to participation.
```

File: content/25.discussion.md [hunk 6]
```diff
@@ -1,37 +1,45 @@
...
+Another technical challenge is that supervised machine learning approaches are neither error free nor bias free.
+By integrating different lines of evidence and preserving uncertainty by analyzing prediction probabilities rather than applying a hard assignment for each prediction, we aimed to alleviate method-specific biases and discover insightful findings that correctly reflect the current representation diversity at conferences.
+
+A key ethical challenge with retrospective work to examine disparities is that algorithmic approaches to infer characteristics are often required.
+This leads to significant limitations such as considering gender as a binary variable.
+While this situation limits retroactive examination, there is a substantial opportunity for scientific societies, grant-making organizations, publishers, and others to proactively collect self-identified demographic information.
+A key element is then implementing processes to use these plans to continuously evaluate practices.
```
Comment (trangdata) [inline content/25.discussion.md]: I'm trying to find a way to say this right and avoid repetitions. Here's a suggestion but can be adjusted.

```suggestion
It is equally crucial that this information is properly used to continuously evaluate inclusion practices.
```

File: content/25.discussion.md [hunk 7]
```diff
@@ -1,37 +1,45 @@
...
+
+A challenge that straddles the ethical and technical divide for studies examining the representation of honorees is that the background that can be best assessed, which we assess here, is the current field composition.
```
Comment (trangdata) [inline content/25.discussion.md]: We used "challenge" to open the last three paragraphs. How about "difficulty"?

File: content/25.discussion.md [hunk 8]
```diff
@@ -1,37 +1,45 @@
...
 Meanwhile, a review of women's underrepresentation in math-intensive fields argued that today's underrepresentation is not explained by historic forms of discrimination but factors surrounding fertility decisions and lifestyle choices, whether freely made or constrained by biology and society [@doi:10.1073/pnas.1014871108].
 A recent analysis of gender inequality across different disciplines showed that, although both gender groups have equivalent annual productivity, women scientists have higher dropout rates throughout their scientific careers [@doi:10.1073/pnas.1914221117].
-Therefore, although we found that ISCB's honorees and keynote speakers appear to have similar gender proportion to the field as a whole, the gender proportions have not reached parity due to these systemic inequalities that obstruct women's progress in science.
+Therefore, although we found that ISCB's honorees and keynote speakers appear to have similar gender proportion to the field as a whole, the gender proportions have not reached parity.
+To the extent that this is due to systemic barriers, the process may have reached equality but not equity.
```
Comment (trangdata) [inline content/25.discussion.md]:
```suggestion
To the extent that this gap is due to systemic barriers, the process may have reached equality but not equity.
```

**Review by trangdata — APPROVED** [review]
Thank you so much for your work on this! 🙏🏽 

I have some minor comments, but please feel free to merge when you're ready.

File: content/25.discussion.md (hunk 7 above)
Comment (cgreene) [reply content/25.discussion.md]: Yes - would accept a suggestion but will also try to remember to come back to this line :)

File: content/metadata.yaml [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Analysis of ISCB honorees and keynotes reveals disparities"
+title: "Analysis of ISCB honors and computational biology authorships reveals disparities"
```
Comment (cgreene) [inline content/metadata.yaml]:
```suggestion
title: "Analysis of ISCB honors reveals disparities"
```

File: content/25.discussion.md (hunk 7 above)
Comment (cgreene) [inline content/25.discussion.md]:
```suggestion
A difficulty that straddles the ethical and technical divide for studies examining the representation of honorees is that the background that can be best assessed, which we assess here, is the current field composition.
```

**Comment by cgreene** [comment]
Ok - I made an additional change to the title to bring it back to a shorter level. See if that looks ok. Thanks!

File: content/02.introduction.md [hunk 10]
```diff
@@ -20,9 +20,10 @@ Alternatively, reviewing scientists may not recognize the scientific importance
...
-We also examined the last authors for more than 170,000 computational biology publications to establish a field-specific baseline using the same metrics.
-The results were consistent across all approaches: we found a dearth of non-White speakers and honorees.
-The lack of Asian scientists among keynote speakers and Fellows was particularly pronounced when compared against the field-specific background.
+We scraped more than 700,000 entries from English-language Wikipedia that contained nationality information and built machine learning classifiers to predict name origin.
+We also examined last authorships for more than 170,000 computational biology publications to establish a field-specific baseline using the same metrics.
+We used methods to predict the gender and name origins of honor recipients and also examined the affiliations of authorships and honor recipients.
+Analysis of affiliations revealed disparities between authorships and honoree affiliations.
+Affiliation analysis revealed disparities in representation by country.
```
Comment (trangdata) [inline content/02.introduction.md]:
```suggestion
Analysis of affiliations by country revealed disparities between authorships and honoree affiliations.
```

File: content/20.results.md [hunk 11]
```diff
@@ -1,63 +1,65 @@
...
 
-Selecting an optimal study design is the important first step to acquire an accurate and thorough answer to any research question.
-This study is not an exception: we were faced with multiple decisions at many analysis steps.
-We are grateful to all the reviewers whose input made the study more rigorous after each revision.
-As we discussed and revised the analysis, we committed to stay transparent and documented the reasoning and potential caveats of each decision.
-Details of these discussions can be found under the [Issues page](https://github.com/greenelab/iscb-diversity-manuscript/issues?q=is%3Aissue+is%3Aclosed) on the study GitHub repository.
-We highlighted here four important decisions we made and how each one may affect the result (Fig. {@fig:versions}).
+Defining an appropriate background distribution is key to determining the extent to which representation diverges from that background.
+We were faced with multiple decisions as we sought to define a field-specific distribution.
+We focused on four key decisions, and have performed analyses in three different ways across those decisions (Fig. {@fig:versions}).
```
Comment (trangdata) [inline content/20.results.md]:
```suggestion
We focused on four key decisions and have performed analyses in three different ways across those decisions (Fig. {@fig:versions}).
```

File: content/20.results.md [hunk 12]
```diff
@@ -1,63 +1,65 @@
...
+Our findings across different versions of the study were consistent with respect to the broad conclusions, though the  numerical results differed.
```
Comment (trangdata) [inline content/20.results.md]:
```suggestion
Our findings across different versions of the study were consistent with respect to the broad conclusions, though the numerical results differed.
```

File: content/20.results.md [hunk 13]
```diff
@@ -1,63 +1,65 @@
...
+and the results were consistent with our findings in [version 2](https://greenelab.github.io/iscb-diversity-manuscript/v/6bda862974e77beb98697f39c18027683af158cb/), where we expanded the article dataset to include all PubMed articles with the MeSH terms "computational biology".
+Finally, in [this version](https://greenelab.github.io/iscb-diversity-manuscript/), we used citation count to weight articles to account for the impact variation of research contributions.
 ](https://raw.githubusercontent.com/greenelab/iscb-diversity/a61d67806a5212488b4c574078f80467cbd8cf5d/figs/versions.png){#fig:versions width="90%"}
 
-Our findings across different versions of the study were generally consistent.
-We first expanded the article set from 29,755 journal-specific to 176,110 all computational biology PubMed articles, making the sample size nearly six times larger than previously.
+In our first version of the analysis pipeline we sought to characterize the distribution of authorships in the field using field-specific journals.
```
Comment (trangdata) [inline content/20.results.md]:
```suggestion
In our first version of the analysis pipeline, we sought to characterize the distribution of authorships in the field using field-specific journals.
```

File: content/20.results.md [hunk 14]
```diff
@@ -1,63 +1,65 @@
...
+This resulted in an analysis set of 29,755 authorships.
+We also examined corresponding authors, as we considered that senior authors may occasionally occupy a different position [@doi:10.1177/0306312716650046], and only fell back to last authors in cases where corresponding author annotations were unavailable.
+
+In the next version of the analysis, we extended the analysis to all 176,110 computational biology PubMed articles substantially increasing the sample size.
```
Comment (trangdata) [inline content/20.results.md]:
```suggestion
In the next version of the analysis, we extended the analysis to all 176,110 computational biology PubMed articles, substantially increasing the sample size.
```

File: content/20.results.md [hunk 15]
```diff
@@ -1,63 +1,65 @@
...
+We present this version of the results in this paper, referring to supplementary figures for other versions of the analyses where warranted.
 
-Finally, in all versions of the analysis, rather than applying a hard assignment for each prediction, we analyzed the raw prediction probability values to capture the uncertainty of the prediction model, improving the accuracy of the aggregate statistics.
-Although our final estimates of the effect size may be conservative (see more in Discussion), the general conclusion remained robust throughout different sets of study design parameters.
-If other researchers aim to answer similar questions of their field, our recommendation is to take caution when making these decisions and account for field-specific factors such as authorship practice.
+Finally, in all versions of the analysis, rather than applying a hard assignment for each prediction, we analyzed the raw prediction probability values to capture the uncertainty of the prediction model.
+Although we expect our estimates of disparities for citation-weighted analyses to be conservative, through each analysis the overall findings remained consistent.
```
Comment (trangdata) [inline content/20.results.md]:
```suggestion
Although we expect our estimates of disparities for citation-weighted analyses to be conservative, through each analysis, the overall findings remained consistent.
```

File: content/20.results.md [hunk 16]
```diff
@@ -67,33 +69,33 @@ Interaction terms did not predict the group of scientists over and above the mai
...
 The two groups of scientists did not have a significant association with names predicted to be European and in Other categories (_p_ = 0.37 and _p_ = 0.44, respectively).
 Interaction terms did not predict the group of scientists over and above the main effect of name origin probability and year.
 
 ![(A) Estimated composition of name origin prediction over the years of
-  all Pubmed computational biology and bioinformatics journal authors (left),
-  and all ISCB Fellows and keynote speakers (right).
-  (B) For each region, the mean predicted probability of Pubmed articles is shown as teal generalized additive model curve, and the mean probability and 95% confidence interval of the ISCB honoree predictions are shown as dark circles and vertical lines.
-  Compared to the name collection of Pubmed authors, honorees with Celtic/English names are overrepresented while honorees with East Asian names are underrepresented.
-  No statistically significant difference was observed between the proportion of honorees and authors with European names or in other categories (see Table @tbl:example_names).
+  all Pubmed computational biology and bioinformatics journal authorships (left),
```
Comment (trangdata) [inline content/20.results.md]:
```suggestion
  all PubMed computational biology and bioinformatics journal authorships (left),
```

**Review by trangdata — APPROVED** [review]
Wow, nicely done! This is a lot of work!!! 🏆  I suggested some final minor fixes.

**Pull request merged**